AcademicReward/Views/GroupsPage.xaml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="AcademicReward.Views.GroupsPage"
+             Title="Your Groups">
+
+    <ContentPage.Resources>
+        <Style TargetType="Button">
```

Comment:
Does this apply to all buttons on the page then? If so, I'm stealing this

Reply:
Yes! That's kinda what got me thinking about styling again on Discord. I hope there's a way to do this project wide (I'm sure there is) or at least a way to import a massive style sheet on every page.